common.mk

```diff
@@ -117,7 +121,13 @@ help: ## Display this help (default)
 ##@ Tooling Binaries
 ## --------------------------------------
 
-tools: $(TOOLING_BINARIES) ## Build tooling binaries
+check_tools:
```

Comment:
If you want this new target to appear in the output of `make help` you need to suffix it with a `##` command with the text to use in the help.  Then again, you may not want to make it an officially public target.

Reply:
doesnt really need to be officially public, ok if i resolve ?

Reply:
Sure. I'll mark as resolved